Okta.Samples.OpenIDConnect.Console/KeyGenerator.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Okta.Samples.OpenIDConnect
+{
+    public class KeyGenerator
+    {
+        public static string GetUniqueKey(int maxSize)
+        {
+            char[] chars = new char[62];
```

Comment:
`char[] chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890".ToCharArray();` is good enough.